docs/content/preview/drivers-orms/rust/diesel.md

```diff
@@ -12,30 +11,45 @@ menu:
 type: docs
 ---
 
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="/preview/drivers-orms/rust/diesel/" class="nav-link active">
+      <i class="fab fa-rust" aria-hidden="true"></i>
+      Diesel
+    </a>
+  </li>
+</ul>
+
 [Diesel](https://diesel.rs/) is a safe, extensible object-relational mapping (ORM) tool and query builder for [Rust](https://www.rust-lang.org/). Diesel lets you create safe and composable abstractions over queries, and eliminates the possibility of incorrect database interactions at compile time. It's designed to be abstracted over, enabling you to write reusable code and think in terms of your problem domain.
 
-YugabyteDB's YSQL API is fully compatible with Diesel ORM for data persistence in Rust applications. This page provides details for getting started with Diesel ORM for connecting to YugabyteDB.
+YugabyteDB's YSQL API is fully compatible with Diesel ORM for data persistence in Rust applications.
+
+## CRUD operations
+
+Learn how to establish a connection to a YugabyteDB database and begin basic CRUD operations using the steps on the [Build an application](../../../quick-start/build-apps/rust/ysql-diesel/) page under the Quick start section.
```

Comment:
all pages - Build an application no longer under Quick Start
"using the steps in [Build an application]."